packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -105,11 +145,22 @@ class IsometricTileMapComponent extends PositionComponent {
 
   /// Same as getBlockRenderPosition but the arguments are exploded as integers.
   Vector2 getBlockRenderPositionInts(int i, int j) {
-    final halfTile = effectiveTileSize / 2;
+    final halfTile = Vector2(
+        effectiveTileSize.x / 2, (effectiveTileSize.y / 2) / scalingFactor);
     final pos = Vector2(i.toDouble(), j.toDouble())..multiply(halfTile);
     return cartToIso(pos) - halfTile;
   }
 
+  /// Same as [getBlockRenderPositionInts] but is used for props ie. elements with <= -2 values
```

Comment:
```suggestion
  /// Same as [getBlockRenderPositionInts] but it is used for props ie. elements with <= -2 values
```

No idea what you mean with `ie` so maybe we should not use abbrevations?

Reply:
by ie. I meant 'in other words' as `prop` itself might not be known to the person reading this. I will change it to full sentence instead.